mycroft/util/format.py

```diff
@@ -142,11 +142,16 @@ def date_format(self, dt, lang, now):
                 if dt.month == now.month and dt.day > now.day:
                     format_str = 'date_full_no_year_month'
 
-            if (now + datetime.timedelta(1)).day == dt.day:
+            tomorrow = now + datetime.timedelta(1)
+            yesterday = now - datetime.timedelta(1)
+            if (tomorrow.day == dt.day and tomorrow.month == dt.month and
```

Comment:
I think this can be simplified by using
```python
if tomorrow.date() == dt.date():
```

The same goes for the ones below

Reply:
Yep, much cleaner